packages/terra-signature/tests/jest/Signature.test.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import Signature from '../../src/Signature';
+
+describe('Signature', () => {
+  const defaultRender = <Signature />;
+
+  // Snapshot Tests
+  it('should render a default component', () => {
+    const signature = shallow(defaultRender);
+    expect(signature).toMatchSnapshot();
+  });
+
+  // Structure Tests
+  it('should have the class terra-Signature', () => {
+    const signature = shallow(defaultRender);
+    expect(signature.prop('className')).toContain('terra-Signature');
```

Comment:
Is this test needed? I know we brought it up before, but it feels like our snapshot tests check this behavior for us.